### ISSUE TITLE
Latex/Mathjax content lost, backslashes get eaten

### ISSUE THREAD

**Issue by dwollsch** [BlueGreenMagick/image-style-editor#13]
Hi,

tldr; field being evaluated causes the contents to be altered. Changes from \t into a tab, and \f into an arrow, etc.

I love your addon! But I just noticed an issue on Anki version:
Version 2.1.22 (0ecc189a)
Python 3.8.0 Qt 5.14.1 PyQt 5.14.1

If I have a mathjax equation then resizing the image will cause any of the back slashed sequences to be removed.

Simple test:
Before setting the image format:
![15-17-3-002324](https://user-images.githubusercontent.com/46449969/111236830-fff64500-85b0-11eb-82bd-e5099d862ad1.jpg)

After setting the image format:
![15-17-3-002325](https://user-images.githubusercontent.com/46449969/111236836-02f13580-85b1-11eb-9d66-5d202dc0820c.jpg)

Note the removal of the backslashes from the field which destroys the mathjax capability. \frac turns into rac, \\( turns into (, and \\) turns into ).

I haven't looked deep into the python but anything that reads in the field contents and then prints it back will probably eat the backslash thinking the user was trying to escape a character or create a tab (\t) or something.

Edit:
So I noticed another thing, the backslashes are not really being "eaten" so to speak. They're just being evaluated to be a new escaped char, for example here's the \t

![image](https://user-images.githubusercontent.com/46449969/111238317-13ef7600-85b4-11eb-8796-8d9e85c6a63e.png)

And we can see where the \f went here in the preview:
![15-17-26-002326](https://user-images.githubusercontent.com/46449969/111238352-25d11900-85b4-11eb-84e7-f3cc7c3d0479.jpg)

**Comment by BlueGreenMagick** [comment]
Fixed thanks to Hwiet!

**Comment by dwollsch** [comment]
Thank you for your time to fix this!!
